contracts/refs/CoreRefV2.sol

```diff
@@ -8,7 +8,7 @@ import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import {IGRLM} from "../minter/IGRLM.sol";
 import {VoltRoles} from "./../core/VoltRoles.sol";
 import {ICoreRefV2} from "./ICoreRefV2.sol";
-import {IPCVOracle} from "./../pcv/morpho/IPCVOracle.sol";
+import {IPCVOracle} from "./../oracle/IPCVOracle.sol";
```

Comment:
Could you create a PR with : 
- move the IPCVOracle.sol to oracle folder (file rename like you did)
- remove of IPCVOracle import in MorphoCompoundPCVDeposit
- update of MorphoCompoundPCVDeposit to use CoreRefV2's hooks

and point it to `feat/core-coreref-cleanup` ?